IdentityCore/src/MSIDConstants.h

```diff
@@ -74,6 +74,28 @@ typedef NS_ENUM(NSInteger, MSIDPromptType)
     MSIDPromptTypeDefault = MSIDPromptTypePromptIfNecessary
 };
 
+typedef NS_ENUM(NSInteger, MSIDAuthScheme)
+{
+    MSIDAuthSchemeBearer,
+    MSIDAuthSchemePop,
+};
+
+typedef NS_ENUM(NSUInteger, MSIDHttpMethod)
```

Comment:
will we support all of those? is there a ready enum from Apple HTTP perhaps?

Reply:
I added all methods which Android supports with its implementation. Will check if Apple provides an inbuilt enum for this.